### ISSUE TITLE
azureSAGetKey fails for providers/Microsoft.ClassicStorage/storageAccounts/mcc360prdstorage

### ISSUE THREAD

**Issue by glglgithub** [Microsoft/AzureSMR#84]
I got the following error message:
 
Fetching Storage Key..
Error: azureSAGetKey()
The Resource 'Microsoft.Storage/storageAccounts/mcc360prdstorage'
under resource group 'Default-Storage-SoutheastAsia' was not
found.
Return code: 404
 
 
However, from resource list, I found that the storage should be under:
/subscriptions/8291f806-7141-425b-***/resourceGroups/Default-Storage-**a/providers/Microsoft.ClassicStorage/storageAccounts/mcc360prdstorage

**Comment by andrie** [comment]
We don't support any of the classic resources at the moment, sorry.

Pull requests for additional functionality are welcome.

**Comment by glglgithub** [comment]
yes, classic resources support is highly required. The classic resources storage has been used as data lake where many data feeds have been dumping to for 3 yrs already. It's impossible to start a new storage, shipped the old storage and re-direct every data stream to the new one. 

Can you instruct how to pull request?
